terraform-provider-project/deployment/provider-test/terraform.tfvars

```diff
@@ -0,0 +1,12 @@
+terragrunt = {
```

Comment:
I would not commit these projects. This structure is coveo specific.

Reply:
Indeed, they should be removed.